src/main/java/nl/hsac/fitnesse/fixture/slim/FileFixture.java

```diff
@@ -49,6 +50,28 @@ public String textIn(String filename) throws IOException {
         return text;
     }
 
+    public String takeFirstLineFrom(String filename) throws IOException {
+        String result;
+        String fullName = getFullName(filename);
+        ensureParentExists(fullName);
+        File file = new File(fullName);
+        Scanner fileScanner = new Scanner(file);
+        if(fileScanner.hasNextLine()) {
+            result = fileScanner.nextLine();
+        } else{
+            throw new IOException(fullName + " is an empty file.");
+        }
+
+        //Overwrite file, minus the first line.
```

Comment:
See commit below. Added a utility method to do an incremental write from the scanner using a printwriter.
This method is used to create a temporary file. Temp file gets copied over the original file and gets deleted afterwards

Is that what you meant?